extensions/resteasy-reactive/jaxrs-client-reactive/deployment/src/main/java/io/quarkus/jaxrs/client/reactive/deployment/ParamData.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.jaxrs.client.reactive.deployment;
+
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.ClassInfo;
+
+/**
+ * ClientxxxParam annotations can be defined on a JAX-RS interface or a sub-client (sub-resource).
+ * If we're filling parameters for a sub-client, we need to know the defining class of the ClientxxxParam
+ * to properly resolve default methods of the "root" client
+ */
+public class ParamData {
```

Comment:
Where's this class used?

Reply:
Sorry again, this was part of another work I was doing and added here by mistake. Removed.

Reply:
No problem :-)